extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -39,10 +39,8 @@ export class AmpAutocomplete extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
-    if (!isExperimentOn(this.getWin(), 'amp-autocomplete')) {
-      user().warn('Experiment %s is not turned on.', EXPERIMENT);
-      return;
-    }
+    userAssert(isExperimentOn(this.win, 'amp-autocomplete'),
```

Comment:
Is there a meaningful difference between user().warn() and userAssert()? I learned why user().warn() was causing trouble in the Travis checks so we can actually undo this change, but I am curious if one way is preferable to the other.